### PR TITLE
Remove striped background from chingu section

### LINKED ISSUE #120: Remove stripe-background from the image in the "What is Chingu?" section
There is a background with diagonal stripes which slipped into the "What is Chingu?" section on the home page. It would be great if it could be removed.

### PR DESCRIPTION
**Pull request by IcemanEtika**
---
name: Remove striped background from chingu section
---

This PR removes the striped background that was in the chingu section of the front page.

Resolves #120

### CODE REVIEW

File: client/components/frontpage/ChinguSection.js [hunk 1]
```diff
@@ -107,7 +106,6 @@ const ChinguSection = () => {
       <ContentWrapper>
         <ImageSection>
           <Logo src="/home-chingu-image.png" />
```
Comment (cMikolai) [inline client/components/frontpage/ChinguSection.js:108]: While you're on it, could you please rename this component? The picture we use is not a Logo, so it should ideally have a different naming.

**Review by cMikolai — CHANGES_REQUESTED** [review]
Hey, 

I'd like to ask for a few changes to clear this component up a bit. I left a comment in the code about the naming of a component and in addition to that:

- Would you be able to please add some padding on tablet sized screen widths for the image on the left?

Bildschirmfoto 2020-10-31 um 15 42 00

And, on another note, please leave a comment under the issue next time when you want to pick it up, so we can assign you to the task and everyone else knows it's work in progress :)

File: client/components/frontpage/ChinguSection.js (hunk 1 above)
Comment (IcemanEtika) [reply client/components/frontpage/ChinguSection.js]: Hey, I just renamed the component and added some padding to the image, please take a look when you have time. And I'll keep in mind to comment under issues before I do them in the future, sorry about that.

Comment (cMikolai) [reply client/components/frontpage/ChinguSection.js]: No problem, thank you!

**Review by cMikolai — APPROVED** [review]
Good job, thank you!

**Review by silentkdev — APPROVED** [review]
thank you so much for the contribution!

**Pull request merged**